src/screens/Auth/LoginScreen/LoginScreen.tsx

```diff
@@ -2,15 +2,50 @@ import React from 'react'
 import {
   Text,
   View,
+  Alert,
+  Image,
+  TouchableOpacity
 } from 'react-native'
 import styles from './styles'
+import BlackHeading from '../../../shared_components/BlackHeading/BlackHeading';
+import GreyTextInput from '../../../shared_components/GreyTextInput/GreyTextInput';
+import PinkButton from '../../../shared_components/PinkButton/PinkButton';
+import PropTypes from 'prop-types'
+
+export default function LoginScreen(props) {
+  const {navigate} = props.navigation;
 
-export default function LoginScreen() {
   return (
     <View style={styles.container}>
-      <Text style={styles.title}>LoginScreen</Text>
+      <View style={styles.main}>
+        <BlackHeading title="Welcome Back" />
+        <Image style={styles.image} resizeMode='contain' source={require('../../../../assets/images/gladeo_logo.png')} />
+        <GreyTextInput placeholder="Email Address" inputType='emailAddress'/>
+        <GreyTextInput placeholder="Password" inputType='password'/>
+        <TouchableOpacity
+          onPress={() => navigate('PasswordReset')}
+        >
+          <Text  style={styles.textButton}>Forgot Password?</Text>
+        </TouchableOpacity>
+        <PinkButton title="LOG IN" onPress={() => Alert.alert('pressed')} />
+      </View>
+
+      <View style={styles.footer}>
+        <View style={styles.subFooter}>
+          <Text style={styles.text}>Don't have an accout?</Text>
+          <TouchableOpacity
+            onPress={() => Alert.alert('pressed')}
+          >
+            <Text style={styles.pinkTextButton}> Sign Up</Text>
+          </TouchableOpacity>
+        </View>
+      </View>
     </View>
   )
 }
 
-
+LoginScreen.propTypes = {
+  navigation: PropTypes.shape({
+    navigate: PropTypes.func.isRequired,
+  }).isRequired,
+}
```

Comment:
Typescript actually replaces the need for propTypes with Interfaces. Because Typescript is designed around type-checking, if you use it correctly, you shouldn't ever need propTypes anymore. You should be doing something like this instead:

```js
interface Props {
  navigation: NavigationScreenProp<NavigationState>;
}

export default function LoginScreen(props: Props)
```

And then you'll need to import `NavigationScreenProp` and `NavigationState` from react-navigation

Read about it here: https://reactnavigation.org/docs/en/next/typescript.html